### DISCUSSION TITLE
What tools do you like and use?

### DISCUSSION THREAD

**Discussion by reubenalfred** [Megabit/Blazorise#4838] [General]
Just out of curiosity what software you like and use and other software you like?

**Comment by stsrki** [comment]
From the developer's side:

Visual Studio
VS Code
SQL Server

Office and web:

Chrome
Thunderbird
Teams
Discord

There are many others but not worth mention I guess.

**Reply by reubenalfred** [reply]
Everything is worth mentioning ☺️
